src/makeCalculator.js

```diff
@@ -3,8 +3,37 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operation, x) {
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+      }
```

Comment:
[CODE KNOWLEDGE] The `divide` method should handle division by zero by not performing the operation, but it should not contain an empty `else` block. It's better to remove the `else` block if there's no action to be taken.